src/libraries/System.Runtime.InteropServices.JavaScript/src/CompatibilitySuppressions.Browser.xml

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Runtime.InteropServices.JavaScript.WebWorker</Target>
```

Comment:
So it looks like this is implementation-only API that's only present in one implementation assembly.  Since it's excluded from the ref that means it's not a problem for users.  Make the suppression conditional as well so that we don't fail the new unused suppressions check.